lib/ui/painting.dart

```diff
@@ -1591,6 +1605,21 @@ enum ImageByteFormat {
   /// image may use a single 8-bit channel for each pixel.
   rawUnmodified,
 
+  /// Raw extended range RGBA format.
+  ///
+  /// Unencoded bytes, in RGBA row-primary form with straight alpha, 32 bit
```

Comment:
Is straight alpha premultiplied or un-premultiplied (post multiplied)?

Reply:
It's un-premultiplied, "straight" is the nomenclature we use in the other fields.